src/main/java/hudson/tasks/junit/JUnitResultArchiver.java

```diff
@@ -241,7 +241,7 @@ public double getHealthScaleFactor() {
      *
      * @since 1.2
      */
-    @DataBoundSetter public final void setTestDataPublishers(@Nonnull List<? extends TestDataPublisher> testDataPublishers) {
+    @DataBoundSetter public final void setTestDataPublishers(@Nonnull List<TestDataPublisher> testDataPublishers) {
```

Comment:
Not source-compatible, but this seems to have been a pretty low-priority use of wildcards.